Platform/CoffeelakeBoardPkg/CfgData/CfgData_GpuConfig.yaml

```diff
@@ -95,7 +95,7 @@
       help         : >
                      Enable or disable
       length       : 0x01
-      value        : 0x1
+      value        : 0x0

```

Comment:
The default value change can impact all boards. What about WHL? Just in case, we can keep this and update cflh/cfls delta files.

Reply:
-Each DDC(B,C,D,E ) and HPD(B,C,D,E) signals  are valid for each DDI ports ( B,C,D,E). 
as per EDS doc, U-processor can only support 1 Edp and 2 DDI ports. Hence Port F Hot plug detect (HPD)  may not valid for WHL.

WHL-U CRB comes with 2 Display ports and I have tested both found working with this change.

Reply:
Understood. Thank you for the confirmation.